src/templates/bootstrap/components/fieldset.html

```diff
@@ -0,0 +1,10 @@
+<div [formGroup]="form" class="form-group">
+    <fieldset>
+        <legend *ngIf="component.settings.legend">{{ component.settings.legend }}</legend>
+        <div *ngFor="let control of component.control.controls;">
+            <div *ngFor="let component of component.settings.components">
+                <formio-component [component]="component" [form]="control"></formio-component>
```

Comment:
Need to pass the ```events``` and ```render``` attributes to the ```<formio-component>``` directive. Look at other container templates.